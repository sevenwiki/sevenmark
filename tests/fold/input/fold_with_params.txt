{{{#fold #style="color:red"
[[#bgcolor="blue" || content with param]]
[[normal content]]
}}}