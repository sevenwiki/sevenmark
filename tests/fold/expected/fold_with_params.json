[
  {
    "FoldElement": {
      "location": {
        "start": 0,
        "end": 92
      },
      "common_style": {
        "style": [
          {
            "Text": {
              "location": {
                "start": 17,
                "end": 26
              },
              "content": "color:red"
            }
          }
        ]
      },
      "content": [
        {
          "common_style": {
            "bg_color": [
              {
                "Text": {
                  "location": {
                    "start": 40,
                    "end": 44
                  },
                  "content": "blue"
                }
              }
            ]
          },
          "content": [
            {
              "Text": {
                "location": {
                  "start": 49,
                  "end": 67
                },
                "content": "content with param"
              }
            }
          ]
        },
        {
          "common_style": {},
          "content": [
            {
              "Text": {
                "location": {
                  "start": 72,
                  "end": 86
                },
                "content": "normal content"
              }
            }
          ]
        }
      ]
    }
  }
]